lib/test-runner-process.coffee

```diff
@@ -0,0 +1,31 @@
+Parser = require 'tap-parser'
+TerminalCommandExecutor = require './terminal-command-executor'
+
+module.exports =
+  class TestRunnerProcess
+    constructor: (executor = new TerminalCommandExecutor) ->
+      @terminalCommandExecutor = executor
+      @terminalCommandExecutor.onDataReceived (data) => @addAvaOutput(data)
+      @terminalCommandExecutor.onDataFinished => @endAvaOutput()
+
+    run: (filePath, assertCallback, completeCallback) ->
+      @parser = @getParser()
+      @parser.on('assert', assertCallback)
+      @parser.on('complete', completeCallback)
+
+      #TODO: Fix parsing of folders and files
+      folder = filePath.substring(0, filePath.lastIndexOf("/") + 1);
+      file = filePath.split("/").pop()
+
+      command = "ava #{file} --tap"
```

Comment:
AVA 0.13.0 changed the API regarding how files are added. Ensure you pass them to the `.run()` method: https://github.com/sindresorhus/ava/blob/6d5f3225cc712f10e7a0a8a9c6e40d574fd4b251/test/api.js#L23-L27

> Same result if I use the CLI in a folder different from the one that hosts the file, as:

I can reproduce too. Would you mind opening an issue on AVA?

```
~/dev/chalk master
❯ ava /Users/sindresorhus/dev/acosh/test.js

  ✖ No tests found in ../acosh/test.js, make sure to import "ava" at the top of your test file
```

-

As a dirty workaround, for now you could probably do this https://github.com/sindresorhus/atom-linter-xo/commit/5d85618ad695b9b7142ac4015a6dde6d8709c6f9.